config/routes.rb

```diff
@@ -11,6 +11,7 @@
         via: %i[get post delete],
         as: :destroy_user_session
   match '/api/saml/auth' => 'saml_idp#auth', via: %i[get post]
+  post '/analytics' => 'analytics#create'
```

Comment:
This endpoint shouldn't be exposed unless a feature is enabled to use it.

Reply:
It gets difficult when all the javascript analytics are all flowing through this one analytics endpoint.  The feature flag should essentially be disabling the javascript for platform authenticators but not necessarily the endpoint.